app/mci/cmd/create.go

```diff
@@ -47,15 +48,16 @@ var executeCommand = func(args []string) (string, error) {
 }
 
 // Extracted out here to allow overriding in tests.
-var getClientset = func(kubeconfigPath string) (kubeclient.Interface, error) {
+var getClientset = func(kubeconfigPath, contextName string) (kubeclient.Interface, error) {
 	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
 	if kubeconfigPath != "" {
 		loadingRules.ExplicitPath = kubeconfigPath
 	}
-	loader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, &clientcmd.ConfigOverrides{})
+	loader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, &clientcmd.ConfigOverrides{CurrentContext: contextName})
 
 	clientConfig, err := loader.ClientConfig()
 	if err != nil {
+		glog.Errorf("getClientset: error getting Client Config:", err)
```

Comment:
Use fmt.Printf() if you want this to print by default since glog.Errorf() prints it as a debug message (with timestamp, file name and line number)
Use glog if you want to hide it behind glog.V().